src/features/Home/GetStarted/GetStarted.module.scss

```diff
@@ -82,11 +82,3 @@
   background-image: url(/img/pattern.svg);
   object-fit: contain;
 }
```

Comment:
please don't remove these for now, since we may get dark theme in the future sprints.

Reply:
reverted